### ISSUE TITLE
Run systematic hyperparameter sweep

### ISSUE THREAD

**Issue by NE0NMIKE** [NE0NMIKE/humanoid-controls-challenge#15]
## Task
Systematically tune the most impactful parameters to get the best tracking performance.

## Parameters to Sweep
| Parameter | Values |
|---|---|
| Learning rate | 3e-4, 1e-4, 5e-5 |
| w_tracking | 1.0, 2.0, 5.0 |
| w_action | 0.001, 0.01, 0.1 |
| num_envs | 256, 512, 1024 |

## Steps
- [ ] Run 6-9 training jobs (use tmux for parallel runs)
- [ ] Log all runs to TensorBoard
- [ ] Compare learning curves: `tensorboard --logdir logs/ --port 6006`
- [ ] SSH port-forward to view on laptop: `ssh -L 6006:localhost:6006 <runpod>`
- [ ] Select the best configuration based on final tracking error
- [ ] Document which hyperparameters you chose and why

## Skills
- Systematic hyperparameter tuning
- TensorBoard for experiment comparison
- SSH port forwarding
